src/docs/sphinx/quickstart.rst

```diff
@@ -265,6 +265,9 @@ Building Serac Dependencies on MacOS
 .. warning::
    These instructions are in development, but have been tested for M2 MacBooks.
 
+.. note::
+   View an example host-config for MacOS in ``host-configs/other``.
```

Comment:
I would link directly to it not the folder. I may be putting one in here soon that will confuse people.